samples/matter/lock/Kconfig

```diff
@@ -148,6 +148,19 @@ config NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
 config NCS_SAMPLE_MATTER_SETTINGS_STORAGE_BACKEND
 	default n if !CHIP_WIFI
 
+# Increase the storage capacity if the schedules are enabled with secure storage
+if LOCK_SCHEDULES && NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND
+
+config NCS_SAMPLE_MATTER_SECURE_STORAGE_MAX_ENTRY_NUMBER
+	default 128
+
+config TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE
+	default 3072
+
+config OPENTHREAD_THREAD_STACK_SIZE
```

Comment:
Why does the Thread stack need to be higher while using schedules and secure storage backend? I guess those Application/Matter specific things should not impact Thread stack occupancy.

Reply:
The exact reason remains unknown and would require more debugging. But the hypothesis is that increasing of the secure storage item maximum size (TRUSTED_STORAGE_BACKEND_AEAD_MAX_DATA_SIZE) affects Thread because something related to secure assets in handled on OT stack during commissioning to Matter.

Reply:
So maybe you could put here a small comment? Because in some time we will think "Why did we increase this stack only for Lock" :)

Reply:
Done.